public/app/percona/dbaas/components/DBCluster/XtraDB.service.ts

```diff
@@ -109,7 +97,7 @@ export class XtraDBService extends DBClusterService {
 
   getExpectedResources(dbCluster: DBCluster): Promise<DBClusterExpectedResources> {
     return apiManagement
-      .post<any, Partial<DBClusterPayload>>('/DBaaS/XtraDBCluster/Resources/Get', pick(toAPI(dbCluster), ['params']))
+      .post<any, Partial<DBClusterPayload>>('/DBaaS/PXCCluster/Resources/Get', pick(toAPI(dbCluster), ['params']))
```

Comment:
The same question about any, can we add types in this part of code?

Reply:
Done, thank you!